tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -286,7 +352,22 @@
     return (node.metrics && node.metrics.time)
               ? tf_op_profile.percent(node.metrics.time) : 0;
   },
-  _selectedChanged: function(v) { this.classList.toggle('selected', v); }
+  _selectedChanged: function(v) { this.classList.toggle('selected', v); },
+  /*
+    Returns top K children for the node. If p90 is true, then only children
+    fallen in the top 90th percentile in time are returned. 
+  */
+  _getKChildren: function(node, k, p90) {
+    if(p90 && node.children.length > 0 && node.children[0].metrics){
+      var tot = 0, i = 0;
+      for (; i < Math.min(k, node.children.length); i++) {
+          if(tot >= node.metrics.time * 0.9) break;
```

Comment:
This multiplication only needs to be computed once, right? Maybe move outside the for loop.

Reply:
Done.